cases/management/commands/enrich_ciaa_allegations.py

```
@@ -0,0 +1,1000 @@
+
+        last_status = None
+        last_body_snippet = ""
+
+        for attempt in range(1, MAX_LLM_RETRIES + 1):
+            try:
+                req = urllib.request.Request(
+                    endpoint,
+                    data=body.encode("utf-8"),
+                    headers=headers,
+                    method="POST",
+                )
```

Ruff (0.15.12)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)